src/index.json

```diff
@@ -605,6 +605,51 @@
                     "version": "0.0.2"
                 }
             }
+        ],
+        "signalr": [
+            {
+                "filename": "signalr-0.1.0-py2.py3-none-any.whl",
+                "sha256Digest": "b902a43c7d5d07feffe313fec023e62437988f33e31eba933e5ef9e9bea21757",
+                "downloadUrl": "https://azureclisignalr.blob.core.windows.net/cli-extensions/signalr-0.1.0-py2.py3-none-any.whl",
+                "metadata": {
```

Comment:
You'll have to add the CLI specific metadata  here also otherwise CI will fail.
```
"azext.minCliCoreVersion": "2.0.31",
"azext.isPreview": true
```